backend/apps/owasp/models/award.py

```diff
@@ -0,0 +1,183 @@
+        verbose_name="Name",
+        max_length=200,
+        help_text="Award name/title (e.g., 'Event Person of the Year')",
+    )
+    description = models.TextField(
+        verbose_name="Description", blank=True, default="", help_text="Award description"
+    )
+    year = models.IntegerField(
+        verbose_name="Year",
+        help_text="Year the award was given",
+    )
+
+    # Winner information
+    winner_name = models.CharField(
+        verbose_name="Winner Name",
+        max_length=200,
+        blank=True,
+        default="",
+        help_text="Name of the award recipient",
+    )
+    winner_info = models.TextField(
+        verbose_name="Winner Information",
+        blank=True,
+        default="",
+        help_text="Detailed information about the winner",
+    )
+    winner_image = models.CharField(
+        verbose_name="Winner Image",
+        max_length=500,
+        blank=True,
+        default="",
+        help_text="Path to winner's image",
+    )
+
+    # Optional foreign key to User model
+    user = models.ForeignKey(
+        "github.User",
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name="awards",
+        verbose_name="User",
+        help_text="Associated GitHub user (if matched)",
+    )
+
+    def __str__(self) -> str:
+        """Return string representation of the award."""
+        if self.winner_name:
+            return f"{self.name} - {self.winner_name} ({self.year})"
+        return f"{self.name} ({self.year})"
+
+    @property
+    def display_name(self) -> str:
+        """Get display name for the award."""
+        return f"{self.name} ({self.year})"
+
+    @classmethod
+    def get_waspy_award_winners(cls):
+        """Get all users who have won WASPY awards.
+
+        Returns:
+            QuerySet of GitHub users who have won WASPY awards
+
+        """
+        from apps.github.models.user import User
+
+        return User.objects.filter(awards__category=cls.Category.WASPY).distinct()
+
+    @classmethod
+    def get_user_waspy_awards(cls, user):
+        """Get all WASPY awards for a specific user.
+
+        Args:
+            user: GitHub User instance
+
+        Returns:
+            QuerySet of WASPY awards for the user
+
+        """
+        return cls.objects.filter(user=user, category=cls.Category.WASPY)
+
+    @staticmethod
+    def update_data(award_data: dict, *, save: bool = True):
+        """Update award data from YAML structure.
+
+        Args:
+            award_data: Dictionary containing award data from YAML
+            save: Whether to save the award instance
+
+        Returns:
+            Award instance or list of Award instances
+
+        """
+        if award_data.get("type") == "award":
+            return Award._create_awards_from_winners(award_data, save=save)
+        return None
+
+    @staticmethod
+    def _create_awards_from_winners(award_data: dict, *, save: bool = True):
```

Comment:
This code doesn't look consistent with the rest of models where `update_data` is responsible for a single instance creation/update logic.